### PR TITLE
Bibliography

### PR DESCRIPTION
**Pull request by OrfeasLitos**

### CODE REVIEW

File: common/references.bib [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+
+@article{sybilattack,
+  author	= "Douceur, J. R.",
+  title		= "The Sybil Attack",
+  journal	= "International workshop on Peer-To-Peer Systems",
+  year		= "2002"
+}
+
+@article{dionyziz,
+  author	= "Zindros, Dionysis Stefanos",
```
Comment (dionyziz) [inline common/references.bib]: Eh?

File: fc17/references.bib [hunk 2]
```diff
@@ -0,0 +1,6 @@
+@splncs{loc,
+  author	= "Sanchez W."
+  title		= "Lines of Credit"
+  year		= "2016"
+  volume	= "\url{https://gist.github.com/drwasho/2c40b91e169f55988618#part-3-web-of-credit"
+}
```
Comment (dionyziz) [inline fc17/references.bib]: Shouldn't this just be part of common/references.bib?

**Comment by dionyziz** [comment]
LGTM; please fix 2 review comments and merge.

File: common/references.bib (hunk 1 above)
Comment (OrfeasLitos) [inline common/references.bib]: I saw it in your master thesis, I suppose it's the name of your father? I am removing it.

File: fc17/references.bib (hunk 2 above)
Comment (OrfeasLitos) [inline fc17/references.bib]: It was already part of common/references.bib, I just removed the file. It isn't needed for compilation.

**Pull request merged**